### DISCUSSION TITLE
`You don't have any saved replies yet`  is wrong

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#133089] [Mobile] [answered]
GitHub Mobile beta for Android 

1. Visit https://github.com/settings/replies
2. If you have saved replies, copy them somewhere and then delete all of them
3. Create a saved reply
4. Delete it
5. See `No saved replies yet`
6. Open GitHub Mobile
7. Open an editor view, e.g. this one. 
8. Trigger saved replies

![Screenshot_20240722-081614.png](https://github.com/user-attachments/assets/20885b18-d64a-44f4-80d6-b562a06f5492)

The text says "You don't have any saved replies yet." 

It's wrong. Yet implies "you never had any saved replies". You had one at the end of step 4. 

You don't have any saved replies anymore, but you had one.

The text is also wrong at step 5 for the same reason, but I'm filing tickets against GitHub Mobile today and the text is different, so that's out of scope.

**Comment by iskae** [comment]
Hi @jsoref 👋 

🤔 This one is a tough one to evaluate. We are aiming to inform the user that the user doesn't have any saved replies yet, so the text also indicates that. Imho the text is semantically correct and understandable, but I would love to learn how we can improve this 🙇 

Do you have any suggestions for improvement? Please keep in mind that the text needs to be concise yet understandable 🙇 

Thanks in advance 🙇

**Reply by jsoref** [reply]
Two possibilities:
1. "You don't have any saved replies. [Create a saved reply](https://github.com/settings/replies)" -- this removes the extraneous/potentially incorrect `yet` and adds something helpful
2. "[Create a saved reply](https://github.com/settings/replies)" -- just removing the entire sentence and adding something more helpful would suffice

Note that the title text `Saved replies are empty` already indicates to a user that the don't have any replies -- so you've said the same thing two different ways in the span of two lines (one that was 100% correct and one that was slightly incorrect).

Fwiw, beyond the above, `Saved replies are empty` itself is wrong grammatically -- `Saved replies` is a singular thing (a list of a saved replies), and thus it should be `Saved replies is empty`.

**Reply by jsoref** [reply]
So, a colleague showed me what the GitHub iOS app does, it's totally different:
![iOS](https://github.com/user-attachments/assets/10ad289c-4ee4-438e-900e-257425ca607d)

1. note that the title is "Saved Replies" -- this is much better than "Choose a reply" (when there are no replies)
2. the first line says "No saved replies" which is again much better than "Saved replies are empty." (there's really no reason to include a period in a title)
3. As noted in my requests, they suggest going to github to create saved replies. (it could be better by being a direct link, but...)
4. Note that there's no onscreen keyboard which given that there's no input field makes a lot of sense.

**Reply by iskae** [reply, marked as answer]
🤔 Sure thing! Let me create an improvement task for myself and update these texts 🙇 Thanks a bunch for your suggestion and also comparison with the iOS app 😊
